emoji/src/main/java/com/gluonhq/emoji/DownloadbleEmojiSpriteLoader.java

```diff
@@ -0,0 +1,89 @@
+package com.gluonhq.emoji;
+
+import javafx.scene.image.Image;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.channels.Channels;
+import java.nio.channels.FileChannel;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.CompletableFuture;
+
+public class DownloadbleEmojiSpriteLoader implements EmojiSpriteLoader {
+
+    private static final String COMMIT_NUMBER = "063f328d7951cb2e2a6678b06dcbdf8dd599fad7"; // tag 15.0.1
+    private static final String EMOJI_PNG_URL = "https://github.com/iamcal/emoji-data/blob/" + COMMIT_NUMBER + "/sheets-clean/sheet_apple_%s_clean.png?raw=true";
+
+    private static final String LOCAL_PATH = System.getProperty("user.home") + "/.gluon/emoji/" + COMMIT_NUMBER;
+    private boolean initialized;
+
+    @Override
+    public boolean isInitialized() {
+        return initialized;
```

Comment:
When you start an application that uses the emoji library, this value is false, and calling `initialize()` takes some milliseconds, long enough to make the process async, and at that point you don't know if there is a need to add some visual feedback for the user (like a progressBar)  or not.

If you add such progress bar, if the check takes 2 milliseconds, you get a visual glitch, because the progress bar is immediately removed.

So I'd say we need a more straightfoward and blocking way to find out if the process is initialised, just by directly checking if the three files exist or not.

Reply:
I have pushed a change to fix this. can you check if the issue is fixed now?